lib/peers.go

```diff
@@ -30,10 +30,7 @@ func (d PeerRequests) CoreRequestsName() string { return "peers" }
 // NewPeerRequests creates a PeerRequests pointer from either a
 // qri Node or an rpc.Client
 func NewPeerRequests(node *p2p.QriNode, cli *rpc.Client) *PeerRequests {
-	if node != nil && cli != nil {
```

Comment:
This check is not needed as it will never conflict. In `PeerRequests.List` we always check if `cli` is present in which case an RPC request is fired and thus result returned and function terminated.